<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width> <!-- Увеличил ширину -->
    <height>700</height> <!-- Увеличил высоту -->
   </rect>
  </property>
  <property name="windowTitle">
   <string>UVM Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout"> <!-- Главный макет - вертикальный -->
    <item>
     <layout class="QGridLayout" name="gridLayout_Panels"> <!-- Макет для панелей SVM -->
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_0">
        <property name="title">
         <string>SVM 0</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Status0">
           <item><widget class="QLabel" name="labelStatus_0"><property name="text"><string>Status:</string></property></widget></item>
           <item><widget class="QLabel" name="valueStatus_0"><property name="font"><font><weight>75</weight><bold>true</bold></font></property><property name="styleSheet"><string notr="true">padding: 2px; background-color: lightgray; color: black;</string></property><property name="text"><string>INACTIVE</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
           <item><widget class="QLabel" name="labelLAK_0"><property name="text"><string>LAK:</string></property></widget></item>
           <item><widget class="QLabel" name="valueLAK_0"><property name="text"><string>0x??</string></property></widget></item>
           <item><widget class="QLabel" name="labelBCB_0"><property name="text"><string>BCB:</string></property></widget></item>
           <item><widget class="QLabel" name="valueBCB_0"><property name="text"><string>0</string></property></widget></item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="errorDisplay_0">
           <property name="text"><string>Status: OK</string></property>
           <property name="styleSheet"><string notr="true">color: green; font-style: italic;</string></property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableLog_0">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_1">
        <property name="title">
         <string>SVM 1</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Status1">
           <item><widget class="QLabel" name="labelStatus_1"><property name="text"><string>Status:</string></property></widget></item>
           <item><widget class="QLabel" name="valueStatus_1"><property name="font"><font><weight>75</weight><bold>true</bold></font></property><property name="styleSheet"><string notr="true">padding: 2px; background-color: lightgray; color: black;</string></property><property name="text"><string>INACTIVE</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
           <item><widget class="QLabel" name="labelLAK_1"><property name="text"><string>LAK:</string></property></widget></item>
           <item><widget class="QLabel" name="valueLAK_1"><property name="text"><string>0x??</string></property></widget></item>
           <item><widget class="QLabel" name="labelBCB_1"><property name="text"><string>BCB:</string></property></widget></item>
           <item><widget class="QLabel" name="valueBCB_1"><property name="text"><string>0</string></property></widget></item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="errorDisplay_1">
           <property name="text"><string>Status: OK</string></property>
           <property name="styleSheet"><string notr="true">color: green; font-style: italic;</string></property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableLog_1">
           <property name="editTriggers"><set>QAbstractItemView::NoEditTriggers</set></property>
           <property name="selectionBehavior"><enum>QAbstractItemView::SelectRows</enum></property>
           <property name="wordWrap"><bool>false</bool></property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>SVM 2</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Status2">
           <item><widget class="QLabel" name="labelStatus_2"><property name="text"><string>Status:</string></property></widget></item>
           <item><widget class="QLabel" name="valueStatus_2"><property name="font"><font><weight>75</weight><bold>true</bold></font></property><property name="styleSheet"><string notr="true">padding: 2px; background-color: lightgray; color: black;</string></property><property name="text"><string>INACTIVE</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
           <item><widget class="QLabel" name="labelLAK_2"><property name="text"><string>LAK:</string></property></widget></item>
           <item><widget class="QLabel" name="valueLAK_2"><property name="text"><string>0x??</string></property></widget></item>
           <item><widget class="QLabel" name="labelBCB_2"><property name="text"><string>BCB:</string></property></widget></item>
           <item><widget class="QLabel" name="valueBCB_2"><property name="text"><string>0</string></property></widget></item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="errorDisplay_2">
           <property name="text"><string>Status: OK</string></property>
           <property name="styleSheet"><string notr="true">color: green; font-style: italic;</string></property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableLog_2">
           <property name="editTriggers"><set>QAbstractItemView::NoEditTriggers</set></property>
           <property name="selectionBehavior"><enum>QAbstractItemView::SelectRows</enum></property>
           <property name="wordWrap"><bool>false</bool></property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>SVM 3</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Status3">
           <item><widget class="QLabel" name="labelStatus_3"><property name="text"><string>Status:</string></property></widget></item>
           <item><widget class="QLabel" name="valueStatus_3"><property name="font"><font><weight>75</weight><bold>true</bold></font></property><property name="styleSheet"><string notr="true">padding: 2px; background-color: lightgray; color: black;</string></property><property name="text"><string>INACTIVE</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
           <item><widget class="QLabel" name="labelLAK_3"><property name="text"><string>LAK:</string></property></widget></item>
           <item><widget class="QLabel" name="valueLAK_3"><property name="text"><string>0x??</string></property></widget></item>
           <item><widget class="QLabel" name="labelBCB_3"><property name="text"><string>BCB:</string></property></widget></item>
           <item><widget class="QLabel" name="valueBCB_3"><property name="text"><string>0</string></property></widget></item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="errorDisplay_3">
           <property name="text"><string>Status: OK</string></property>
           <property name="styleSheet"><string notr="true">color: green; font-style: italic;</string></property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableLog_3">
           <property name="editTriggers"><set>QAbstractItemView::NoEditTriggers</set></property>
           <property name="selectionBehavior"><enum>QAbstractItemView::SelectRows</enum></property>
           <property name="wordWrap"><bool>false</bool></property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item> <!-- Элемент для кнопки сохранения, растягивающийся -->
     <widget class="QPushButton" name="buttonSaveAllLogs">
      <property name="text">
       <string>Сохранить все логи</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>